src/components/Switch.js

```diff
@@ -0,0 +1,145 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import defaultTheme from '../themes/DefaultTheme';
+import darkTheme from '../themes/DarkTheme';
+
+const Switch = ({
+  label,
+  position,
+  checked,
+  disabled,
+  onChange,
+}) => (
+  <label
+    className={classnames('switch-wrapper', {
+      [`switch-wrapper--${position}`]: label && position,
+    })}
+  >
+    {label && <span className="switch-label">{label}</span>}
+    <input
+      className="switch-input"
```

Comment:
Kindly, use BEM methodology in this project, for instance `switch-input` --> `switch__input`